src/main/java/com/google/javascript/gents/GentsCodeGenerator.java

```diff
@@ -26,6 +26,11 @@ protected GentsCodeGenerator(
 
   @Override
   protected void add(Node n, Context ctx) {
+    // Other CodeGenerators seem to safely assume n is non-null. So having a null n is
+    // odd. That said, I know this happens for some users and cannot reproduce it yet, so for now
+    // just continue emitting instead of crashing.
+    // TODO(rado): reproduce having a null node and fix at the generator source.
+    if (n == null) return;
```

Comment:
Isn't there some standard way to dump a stack trace?

Reply:
I can get to the null node reproducibly in g3, but because this is during the emit phase it is hard to trace back to site of a actual AST mutation that introduced it.